config/300-clusteringress.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusteringresses.networking.internal.knative.dev
+spec:
+  group: networking.internal.knative.dev
+  version: v1alpha1
+  names:
+    kind: ClusterIngress
+    plural: clusteringresses
+    singular: clusteringress
+    categories:
+    - all
+    - knative-internal
+    - networking
+    shortNames:
+    - king
```

Comment:
I'd suggest you don't want a short name for an "internal" resource -- you don't really want users to find these and play around with them.